Validate P2 license requirements....

CHDAFNI-MSFT opened the pull request:
The statement stating only 1 x AAD P2 license is needed to activate the feature for the tenant sounds grossly misrepresented from a licensing perspective. Similarly, we have other AAD P2 features which are tenant wide features, but our licensing policy dictates a P2 license is needed for each user using it (even though the AAD system doesn't validate which users have P2 licenses).
My point is this... if you're using a single P2 license as a mechanism to activate tenant features to be used by everyone... you're likely violating our license terms (regardless of what the technology allows you to do with 1x license)

azurecla commented:
Hi __@KundanaP__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@KundanaP Please submit new articles through the private repository. See email for more details.

CHDAFNI-MSFT committed:
Validate P2 license requirements....

The statement stating only 1 x AAD P2 license is needed to activate the feature for the tenant sounds grossly misrepresented from a licensing perspective. Similarly, we have other AAD P2 features which are tenant wide features, but our licensing policy dictates a P2 license is needed for each user using it (even though the AAD system doesn't validate which users have P2 licenses).
My point is this... if you're using a single P2 license as a mechanism to activate tenant features to be used by everyone... you're likely violating our license terms (regardless of what the technology allows you to do with 1x license)
